src/hayhooks/server/routers/undeploy.py

```diff
@@ -1,23 +1,22 @@
 from typing import cast
-
-from fastapi import HTTPException
+from fastapi import APIRouter, HTTPException
 from fastapi.routing import APIRoute
-from hayhooks.server import app
 from hayhooks.server.pipelines import registry
 
+router = APIRouter()
+
 
-@app.post("/undeploy/{pipeline_name}", tags=["config"])
-async def deploy(pipeline_name: str):
+@router.post("/undeploy/{pipeline_name}", tags=["config"])
+async def undeploy(pipeline_name: str):
```

Comment:
Was this an obvious mistake before?

Reply:
yes, there was a typo in the existing route ;)